lib/util.js

```diff
@@ -67,6 +73,11 @@ module.exports.kickoff = function kickoff() {
           console.log(results);
           /* eslint-enable */
         }
+        done();
       }.bind(this));
+    }.bind(this);
```

Comment:
agree this could be a change for a follow-on PR. the `bind` stuff is already there so would prefer accomplishing the aim of this PR with minimal other changes.